### PR TITLE
add transformer manager test

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: BaseApiBundle/Tests/Transformer/TransformerTest.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * test TransformerManager class
+     */
+    public function testTransformerManager()
+    {
+        $name = 'first_transfomer';
+        $transformer = Phake::mock('OpenOrchestra\BaseApi\Transformer\TransformerInterface');
+        Phake::when($transformer)->getName()
+            ->thenReturn($name);
+        Phake::when($transformer)->setContext(Phake::anyParameters());
```
Comment (ngilain) [inline BaseApiBundle/Tests/Transformer/TransformerTest.php]: Is this line usefull ? It seems to lack a `thenReturn` call.

Comment (SOLenG) [inline BaseApiBundle/Tests/Transformer/TransformerTest.php]: You take a point, I added this line only to respect all steps of the function addTransformer.

**Comment by nicolasThal** [comment]
:+1:

**Comment by nicolasThal** [comment]
you should rebase the master branch

**Pull request merged**